doc/source/development/release_process.rst

```diff
@@ -0,0 +1,217 @@
+.. Licensed to the Apache Software Foundation (ASF) under one
+.. or more contributor license agreements.  See the NOTICE file
+.. distributed with this work for additional information
+.. regarding copyright ownership.  The ASF licenses this file
+.. to you under the Apache License, Version 2.0 (the
+.. "License"); you may not use this file except in compliance
+.. with the License.  You may obtain a copy of the License at
+..
+..     http://www.apache.org/licenses/LICENSE-2.0
+..
+.. Unless required by applicable law or agreed to in writing, software
+.. distributed under the License is distributed on an "AS IS" BASIS,
+.. WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+.. See the License for the specific language governing permissions and
+.. limitations under the License.
+
+.. highlight:: none
+..  release_process:
+
+Release Process
+***************
+
+.. contents:: :depth: 3
+
+| 
+|
+
+Here Release Managers will find the steps for creating, voting and publishing a release for Apache Cassandra.
+A committer can perform the initial steps of creating and calling a vote on a proposed release, but only a PMC can complete the process of publishing and announcing the release.
+
+
+Prerequisites
+=============
+
+Be familiar with the following
+ * `ASF Release Policy <http://www.apache.org/legal/release-policy.html>`_.
+ * `ASF Release Distribution Policy <http://www.apache.org/dev/release-distribution>`_.
+ * `ASF Release Best Practices <http://www.eu.apache.org/dev/release-publishing.html>`_.
+
+
+Create and publish your GPG key
+-------------------------------
+
+To create a GPG key, follow the `guidelines <http://www.apache.org/dev/openpgp.html>`_.
+Include your public key in::
+
+  https://dist.apache.org/repos/dist/cassandra/KEYS
+
+
+Publish your GPG key in a PGP key server, such as `MIT Keyserver <http://pgp.mit.edu/>`_.
+
+
+Create Release Artifacts
+========================
+
+Any committer can perform the following steps to create and call a vote on a proposed release.
+
+Check that no open jira tickets are urgent and currently being worked on.
+Also check with a PMC that there's security vulnerabilities currently being worked on in private.
+
+Perform the Release
+-------------------
+
+Run the following commands to generate and upload release artifacts, to a nexus staging repository and distribution location::
+
+    git clone https://github.com/apache/cassandra-builds.git
+    cd cassandra-build/cassandra-release
+    # edit the variables at the top of `prepare_release.sh`
+    ./prepare_release.sh
+
+If successful, take note of the email text output which can be used in the next section "Call for a Vote".
+
+The ``prepare_release.sh`` script does not yet generate and upload the rpm and deb distribution packages.
```

Comment:
I think it does build a deb package at least, see [prepare_release.sh#L167](https://github.com/apache/cassandra-builds/blob/master/cassandra-release/prepare_release.sh#L167). This part should probably be replaced in the script by the docker based build process mentioned as follows.

Reply:
Oh!

And yes, IMHO it should be replaced with the docker approach. But I'd also say the building and upload of the artifacts should be done in a reproducible docker image, ie these lines https://github.com/apache/cassandra-builds/blob/master/cassandra-release/prepare_release.sh#L145-L160
There's too many variables allowed there, versions of java, versions of ant, what's in your local .m2/repository, etc etc.